plugin/evm/mempool.go

```diff
@@ -491,3 +524,25 @@ func (m *Mempool) GetNewTxs() []*Tx {
 	m.metrics.newTxsReturned.Inc(int64(len(cpy))) // Increment the number of newTxs
 	return cpy
 }
+
+var _ gossip.Tx = (*MempoolTx)(nil)
```

Comment:
Maybe we should just make `evm.Tx` and `core.Transaction` implement `gossip.Tx` instead of this wrapper pattern